ant/org.eclipse.ant.core/src/org/eclipse/ant/internal/core/contentDescriber/AntBuildfileContentDescriber.java

```diff
@@ -59,7 +61,9 @@ private int checkCriteria(InputSource contents) throws IOException {
 		}
 		catch (SAXException e) {
 			// we may be handed any kind of contents... it is normal we fail to parse
-			AntCorePlugin.log(e);
+			// must not log an error - or test log listener will fail the test
```

Comment:
Could you document the rationale for logging no error but a warning by the fact that the exception thrown here is actually "unexceptional" like mentioned  in the preceeding line ("it is normal we fail to parse") rather than referring to a failing test? The comment currently sounds as if this code is adapted to make some test work but actually the test just revealed that logging an error here may not be reasonable.